src/app/admin/_components/faq/List.js

```diff
@@ -0,0 +1,132 @@
+'use client';
+
+import {
+  Datagrid,
+  List,
+  TextField,
+  WrapperField,
+  useNotify,
+  useRecordContext,
+  useRefresh,
+  useUpdate,
+} from 'react-admin';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
🔴 i think that should be resolved within dependencies

Reply:
Thanks, for comment! Resolved.